src/components/scenes/ChangeMiningFeeScene.js

```diff
@@ -96,6 +108,17 @@ export class ChangeMiningFee extends React.Component<Props, State> {
           {customFormat != null ? this.renderRadioRow('custom', s.strings.mining_fee_custom_label_choice) : null}
           {customFormat != null ? this.renderCustomFee(customFormat) : null}
           {this.renderFeeWarning()}
+          <View>
+            <View style={styles.row}>
```

Comment:
Turn into component which receives the props:

-  label/children
- init value
- on change (function)